rosdep/base.yaml

```diff
@@ -1797,6 +1798,7 @@ libfreetype6:
   ubuntu: [libfreetype6]
 libfreetype6-dev:
   debian: [libfreetype6-dev]
+  fedora: [freetype-devel]
```

Comment:
I believe that the `libfreetype6` key could probably be relaxed to drop the `-devel` suffix some day, but I haven't verified that.